lib/Gateway.js

```diff
@@ -1488,7 +1518,19 @@ Gateway.prototype.discoverValue = function (node, vId) {
         } else {
           cfg.object_id = 'notification_' + valueId.property
         }
-        cfg.discovery_payload.icon = 'mdi:alarm-light'
+        switch (valueId.propertyKey) {
+          case 'Motion sensor status':
+            cfg.discovery_payload.icon = 'mdi:motion-sensor'
+            break
+          default:
+            cfg.discovery_payload.icon = 'mdi:alarm-light'
+        }
+        if (valueId.list) {
+          cfg.discovery_payload.value_template = getMappedStateTemplate(
+            valueId.states,
+            0
```

Comment:
could this be `valueId.default` instead?

Reply:
I did not see this! and i was thinking about it! cool! let me check the payload and fix this! :)

Reply:
```ccSpecific:{notificationType: 7}
commandClass:113
commandClassName:'Notification'
default:undefined
description:undefined
endpoint:0
id:'95-113-0-Home Security-Motion sensor status'
label:'Motion sensor status'
lastUpdate:1608645438998
list:true
max:255
min:0
nodeId:95
property:'Home Security'
propertyKey:'Motion sensor status'
propertyName:'Home Security'
readable:true
states:(2) [{…}, {…}]
0:{text: 'idle', value: 0}
1:{text: 'Motion detection', value: 8}
length:2
__proto__:Array(0)
step:undefined
type:'number'
unit:undefined
value:0
writeable:false
__proto__:Object
```
default is undefined. I think i need to make something like
`valueId.default ? valueId.default : 0`

Reply:
@billiaz I think both are wrong, IMO default should not be a valid state, I mean if the value is not in the list of states with a default we will never know this is happening, we should show something like `Unknown` when the state is unknown

Reply:
in that case home assistant will return the number! and user will see it as a broken setup which is acceptable.
when i post the template with out the `8:` i get 8 instead of motion detection.

The default value on states is always (per documentation) 0 (excel)

Reply:
Ok so I think that is good, not to use a default one that exists